### PR TITLE
Added 24.1 major to release notes

### PR DESCRIPTION
**Pull request by justinegeffen**
- Added release notes
- Fixed incorrect date that was causing display issues

### CODE REVIEW

File: changelog/seqera-enterprise/v24.1_major.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+| `micronaut.security.token.jwt.generator.access-token.expiration` | `micronaut.security.token.generator.access-token.expiration` |
+	
+Enterprise deployments that have customized this value previously will need to adopt the new format.
+
+## Upgrade steps 
+
+1. This version includes an update to the Platform Enterprise H8 cache. **Do not start the upgrade while any pipelines are in a `running` state as active run data may be lost.**
+1. This version requires a database schema update. Make a backup of your Platform database prior to upgrade.
+1. If you are upgrading from a version older than 23.4.1, update your installation to version [23.4.4](https://docs.seqera.io/changelog/seqera-enterprise/v23.4_major) first, before updating to 24.1 with the steps below.
+1. For recommended Platform memory settings, add the following environment variable to your Platform configuration values (`tower.env`, `configmap.yml`, etc. ):
```
Comment (christopher-hakkaart) [inline changelog/seqera-enterprise/v24.1_major.mdx:82]:
```suggestion
1. For recommended Platform memory settings, add the following environment variable to your Platform configuration values (`tower.env`, `configmap.yml`, etc.):
```

**Review by christopher-hakkaart — APPROVED** [review]
I don't understand the date changes but otherwise looks good!

File: changelog/seqera-enterprise/v24.1_major.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
+---
+title: Seqera Enterprise v24.1_major
+date: 2024-07-04
+tags: [seqera enterprise]
+---
+
+Seqera Platform Enterprise version 24.1 introduces three new features: Data Studios (in public preview), Data Explorer, and managed identities. A number of bug fixes and performance enhancements have also been included in this major release.
```
Comment (jason-seqera) [inline changelog/seqera-enterprise/v24.1_major.mdx:7]:
```suggestion
Seqera Platform Enterprise version 24.1 introduces three new features: Data Studios (in public preview), Data Explorer, and managed identities. A number of bug fixes and performance enhancements are included in this major release.
```

File: changelog/seqera-enterprise/v24.1_major.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+## Data Studios
+
+[Data Studios](https://docs.seqera.io/platform/24.2/data_studios) closes the loop from development to deployment and insights, allowing you to create, manage, and share notebook environments in Seqera with the click of a button. Data Studios also makes it seamless to work across teams with multi-user support, built-in authentication, and automatic snapshots as you work.
```
Comment (jason-seqera) [inline changelog/seqera-enterprise/v24.1_major.mdx:11]:
```suggestion
[Data Studios](https://docs.seqera.io/platform/24.2/data_studios) closes the loop from development to deployment and insights, allowing you to create, manage, and share notebook environments in Seqera with the click of a button. Data Studios makes it seamless to work across teams with multi-user support, built-in authentication, and automatic snapshots as you work.
```

File: changelog/seqera-enterprise/v24.1_major.mdx [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+- Host a combination of container images and compute environments for interactive analysis using your preferred tools, like Jupyter notebooks, RStudio, and Visual Studio Code IDEs.
+- Checkpoints provide a stable point-in-time snapshot.
+- Collaborate and share data in real time.
+
+Data Studios is available as a public preview. [Contact support](https://support.seqera.io) to request access to this feature.
```
Comment (jason-seqera) [inline changelog/seqera-enterprise/v24.1_major.mdx:17]: We're using this in docs

```
:::note
Data Studios is currently in **private preview** and is not yet generally available. To request access, please contact your Seqera account executive.
:::
```

File: changelog/seqera-enterprise/v24.1_major.mdx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- Checkpoints provide a stable point-in-time snapshot.
+- Collaborate and share data in real time.
+
+Data Studios is available as a public preview. [Contact support](https://support.seqera.io) to request access to this feature. 
+
+## Data Explorer
+
+[Data Explorer](https://docs.seqera.io/platform/24.2/data/data-explorer) allows you to browse and interact with remote data repositories from organization workspaces in Seqera Platform. It supports AWS S3, Azure Blob Storage, and Google Cloud Storage repositories.
+
+Data Explorer is now Generally Available and supports multi-file and multi-folder download. This allows users to download entire datasets of pipeline results to share with their team.
```
Comment (jason-seqera) [inline changelog/seqera-enterprise/v24.1_major.mdx:23]:
```suggestion
Data Explorer is now Generally Available (GA) and supports multi-file and multi-folder download. This allows users to download entire datasets of pipeline results to share with their team.
```

File: changelog/seqera-enterprise/v24.1_major.mdx [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+## Extended data roles
+
+User roles provide flexibility for admins to provide users with the permissions they need, without compromising security. The **Connect** user role has been added to the existing user roles with functionality related to Data Studios:
+
+- **Connect**: Open a connection to a running data studio session and interact with the contents.
+- **Maintain**: Create, update, start, stop, change configuration, and delete data studio sessions.
```
Comment (jason-seqera) [inline changelog/seqera-enterprise/v24.1_major.mdx:30]:
```suggestion
- **Maintain**: Create, update, start, stop, configure, and delete data studio sessions.
```

File: changelog/seqera-enterprise/v24.1_major.mdx [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+- **View**: List, search, and view the status/configuration/details of data studios.
```
Comment (jason-seqera) [inline changelog/seqera-enterprise/v24.1_major.mdx:31]:
```suggestion
- **View**: List, search, and view the status, configuration, and details of data studios.
```

File: changelog/seqera-enterprise/v24.1_major.mdx [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+## Managed identities
+
+Managed identities offer significant advantages for High-Performance Computing (HPC) environments by enabling granular access control for individual workspace users. Unlike traditional SSH credentials that grant all workspace users access to HPC clusters with the same service account, managed identities ensure each user’s activity is logged under their own credentials.
+
+See [Managed identities](https://docs.seqera.io/platform/24.2/credentials/managed_identities) for more information. 
+
+## Other feature improvements
+
+- A redesigned [pipeline launch form](https://docs.seqera.io/platform/24.2/getting-started/quickstart-demo/launch-pipelines) provides an improved workflow and an interface to create and edit run parameters as raw JSON/YAML. Enable the new launch form per workspace via [environment variable](https://docs.seqera.io/platform/24.2/enterprise/configuration/overview#core-featuress).
```
Comment (jason-seqera) [inline changelog/seqera-enterprise/v24.1_major.mdx:44]:
```suggestion
- A redesigned [pipeline launch form](https://docs.seqera.io/platform/24.2/getting-started/quickstart-demo/launch-pipelines) provides an improved workflow and an interface to create and edit run parameters as raw JSON/YAML. Enable the new launch form per workspace via [environment variable](https://docs.seqera.io/platform/24.2/enterprise/configuration/overview#core-features).
```

Cleaning up the failed anchor validation errors would avoid anymore instances of broken anchors.

**Review by jason-seqera — COMMENTED** [review]
@justinegeffen, some minor comments, thanks!

File: changelog/seqera-enterprise/v24.1_major.mdx (hunk 4 above)
Comment (justinegeffen) [reply changelog/seqera-enterprise/v24.1_major.mdx:17]: Let's update the docs to reflect that it's in public preview.

Comment (jason-seqera) [reply changelog/seqera-enterprise/v24.1_major.mdx:17]: True. But also different wording:

**To request access, please contact your Seqera account executive.**

**Pull request merged**